kolibri/core/device/serializers.py

```diff
@@ -75,7 +78,11 @@ def validate(self, data):
         ):
             data["os_user"] = True
         elif "superuser" not in data:
-            raise serializers.ValidationError("Superuser is required for provisioning")
+            # We don't need a superuser if we're on an SoUD
```

Comment:
Why not? We still need a superuser, otherwise, there is no one who can administer the device. By default the first imported user should be the superuser.